parser.js

```diff
@@ -186,12 +210,15 @@ function generateAST(code, options, additionalParsingContext) {
 exports.version = require("./package.json").version;
 
 exports.parse = function parse(code, options) {
-    return generateAST(code, options, { isParseForESLint: false });
```

Comment:
The name `generateAST` no longer makes sense for this function after this change